src/routes/api/users.js

```diff
@@ -41,5 +42,6 @@ router.post('/requestPasswordReset', Users.requestPasswordReset);
 router.patch('/resetPassword/:userId/:token', userValidator.resetPassword, Users.resetPassword);
 router.post('/forgotPassword', Users.requestPasswordReset);
 router.put('/resetPassword/:userId/:token', userValidator.resetPassword, Users.resetPassword);
+router.put('/updateRole', userValidator.validateUserRole, Access.isAdmin, Users.updateUserRole);
```

Comment:
Since the first middleware decodes the token, can't you remove the part you decode it in the second and access it through `req.user`